device.py

```diff
@@ -380,8 +380,8 @@ def start(self, avd = None):
         else:
             logger.info("starting avd from arg %s" % avd)
         cmd = 'nice emulator -avd %s -debug-init' % (self.avd_inst if avd is None else avd)
-        if self.console_mode:
-            cmd += " -no-window"
+        #if self.console_mode:
+        #    cmd += " -no-window"
```

Comment:
We run the experiments on headless mode, i.e., I don't have X servers running. Do you mean that now emulators can detect this and would not fail?